### PR TITLE
Organizing comments in info tables

### PR DESCRIPTION
**Pull request by felipeaoli**

### CODE REVIEW

**Comment by felipeaoli** [comment]
I kept the @felipelm comments display but I corrected the values and ref values in table.

File: backend/framework/qlf/dashboard/bokeh/qagetrms/main.py [hunk 1]
```diff
@@ -84,10 +84,12 @@ def load_qa(self):
         info_col = Div(text=write_description('getrms'))
 
         # Prepare tables
-        comments = 'value of RMS for each amplifier read directly from the header of the pre processed image'
+        comments = 'Value of RMS for each amplifier read directly from the header of the pre processed image'
         refexp = mergedqa['TASKS']['CHECK_CCDs']['PARAMS']['BIAS_AMP_REF']
-        metric_txt = mtable('getbias', mergedqa)
+        metric_txt = mtable('getrms', mergedqa, comments)
         metric_tb = Div(text=metric_txt)
+        with open("dbgtb.html", 'w') as f:
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/qagetrms/main.py]: This is creating a unnecessary file

Comment (felipeaoli) [reply backend/framework/qlf/dashboard/bokeh/qagetrms/main.py]: lines removed

File: backend/framework/qlf/dashboard/bokeh/qlf_plot.py [hunk 2]
```diff
@@ -161,7 +161,7 @@ def metric_table(metricname, comments, keyname,  curexp='xxx', refexp='xxx', obj
     return style + title + header + tblines + end
 
 
-def mtable(qa, data, objtype=['XXELG', 'XXSTAR']):
+def mtable(qa, data, comments, objtype=['OTYPE ?', 'OTYPE ?']):
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/qlf_plot.py]: Is this `comments` variable necessary?

You are not using it so let's remove it

Comment (felipeaoli) [reply backend/framework/qlf/dashboard/bokeh/qlf_plot.py]: done

**Pull request merged**